code/src/main/webapp/JavaScript/script.js

```diff
@@ -170,24 +170,13 @@ function attachSignin(element, negativeUser) {
     document.getElementById('name').innerText = "Signed in: " + googleUser.getBasicProfile().getName();
 
     const idToken = googleUser.getAuthResponse().id_token;
+    console.log(idToken.toString());
+    localStorage.setItem('idToken', idToken.toString());
     const params = new URLSearchParams()
     params.append('idToken', idToken);
-    const request = new Request('/authenticate', {method: 'POST', body: params});
-    fetch(request).then(() => {
-      if (negativeUser) {
-        PAGE_CONTROLLER.show('notification')
-      }
-    });
+    fetch(new Request('/authentication-test', {method: 'POST', body: params})).then(response => response.text()).then(url => window.location = url);
```

Comment:
Is this just a test? I'm just really confused about this.

Reply:
Yep, just to test that it works